tile.yml

```diff
@@ -154,8 +140,28 @@ forms:
     label: PCF foundation name
     description: A user friendly PCF foundation name. This is included as a point tag on the metrics sent to Wavefront.
     optional: false
-  - name: pcf_skipSslValidation
+  - name: pcf_skip_ssl_validation
     type: boolean
     label: Skip SSL Verification
     default: false
     description: Useful for testing environments where SSL is not properly setup. Should not be used in prod environments.
+  - name: filter_metrics_white_list
```

Comment:
Does this env variable name map correctly to the nozzle? I see `MetricsBlackList` etc in the Nozzle.

Reply:
yes, this will be translate to a env var called `FILTER_METRICS_WHITE_LIST` and that is put into  `MetricsWhiteList` (it have the `split_words:"true"` tag)